sdks/python/apache_beam/yaml/examples/test/regex_matches_yaml_test.py

```diff
@@ -0,0 +1,41 @@
+# coding=utf-8
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# pytype: skip-file
+
+import unittest
+
+from apache_beam.yaml.examples.test.test_yaml_example import test_yaml_example
+
+YAML_FILE = '../regex_matches.yaml'
+
+EXPECTED = [
```

Comment:
Actually, it looks like these are all *very* formulaic. 

What if we put a comment 

`# Expected: ...`

right in the yaml file itself? This would be useful to the reader of the yaml file, and would obviate the need for all these separate test files (we would just do a glob for *.yaml in the directories). 

We could alternatively add an "expected" parameter to LogForTesting (which would do its verification via AssertThat, no need to even scrape stdout) if we didn't want to parse it out ourselves.

Reply:
I like the idea of putting the `# Expected` tag in the yaml file so that a user can see what the expected output is easily.